src/main/java/com/kine/Kine/food/controller/FoodController.java

```diff
@@ -0,0 +1,102 @@
+package com.kine.Kine.food.controller;
+import com.kine.Kine.food.dto.CreateFoodDTO;
+import com.kine.Kine.food.dto.GetFoodDTO;
+import com.kine.Kine.food.dto.UpdateDTO;
+import com.kine.Kine.food.exception.*;
+import com.kine.Kine.food.service.FoodService;
+import com.kine.Kine.food.dto.FoodDTO;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.data.domain.Pageable;
+
+
+
+@RestController
+@RequestMapping("/api/food")
+public class FoodController {
+    private final FoodService foodService;
+    public FoodController(FoodService foodService){
+        this.foodService = foodService;
+    }
+
+
+    @GetMapping(value = "/{id}")
+    public ResponseEntity<?> searchFoodById(@PathVariable(value = "id") Long id) {
```

Comment:
call it getFoodById, you are retreiving not searching

Reply:
done

Reply:
no it's not

Reply:
![image](https://github.com/oussamaty/kine-back/assets/134633040/3ccc007e-3d40-40c6-8ec2-2a159c2d9927)

Reply:
bah sii